eval/testdata/iface2_error.ng

```diff
@@ -0,0 +1,7 @@
+import "io"
+
+w := io.Writer(nil)
+r := io.Reader(nil) // ERROR: cannot assign 
```

Comment:
Interesting, the error message is off-by-one line? Want to file an issue to track that?

Reply:
ah! you're right.
I suppose it's the test harness that doesn't check the correct line number...
I'll file an issue.